xml/System.Security.Cryptography/AesManaged.xml

```diff
@@ -690,8 +706,12 @@
       <Docs>
         <summary>Gets or sets the padding mode used in the symmetric algorithm.</summary>
         <value>One of the enumeration values that specifies the type of padding to apply. The default is <see cref="F:System.Security.Cryptography.PaddingMode.PKCS7" />.</value>
-        <remarks>To be added.</remarks>
+        <remarks><format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ Padding is used when there's not enough data to fill the last block. See the <xref:System.Security.Cryptography.PaddingMode> enumeration for a description of specific modes.  
```

Comment:
Seems like this should re-use the value from the SymmetricAlgorithm base class, or just seealso to SymmetricAlgorithm's version of it.